src/components/user/EditDialog.vue

```diff
@@ -107,6 +113,7 @@ export default {
           }
         })
         .catch((error) => {
+          this.$alert("Update user error", "success:false");
```

Comment:
Questo visualizza effettivamente il messaggio di errore con la croce rossa? Secondo me devono essere passati 3 argomenti all'alert, il test, il titolo e il tipo: in questo caso dovrebbe mancare il titolo in mezzo.